frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx

```diff
@@ -350,4 +351,4 @@ const showStateUpdateMessage = (
 	}
 
 	message.success({ content: displayMessage, key: MESSAGE_KEY }, 10)
-}
+}
```

Comment:
@d29107d - your editor is removing the newline here and that's why the build is failing.
Per what @Vadman97 said, can you run `yarn format:all` and commit the change?

Reply:
Done. 
@et could you help to rerun the CI?

Reply:
Yup. Just ran it.

Reply:
@et I don't know why the CI failed so I rebase the main branch, it needs you to rerun the CI. 🙏

Reply:
The error seems like it might have been related to #4703. Rebasing definitely seems like the right call.
Just re-ran. Hopefully this one passes 🤞

Reply:
Actually. It looks like we might have to wait until #4734 is merged through. Will let you know when that happens. Thanks for your work on this @d29107d

Reply:
👌